anchor/message_validator/src/lib.rs

```diff
@@ -248,8 +248,9 @@ impl<S: SlotClock> Validator<S> {
                             .ok_or(ValidationFailure::UnknownValidator)?
                     }
                 };
-
-                let operators_pks = self.get_operator_pks(signed_ssv_message.operator_ids())?;
+                let operators_pks =
+                    get_operator_pks(&network_state, signed_ssv_message.operator_ids())?;
+                drop(network_state);
```

Comment:
That's a good catch, thanks. Is there a way we can avoid repeating this in the future? It still seems error-prone.